frontend/tests/e2e/member/filter/active-on.js

```diff
@@ -0,0 +1,63 @@
+export default () => {
+  before(() => {
+    cy.wait(1000);
```

Comment:
I'm not super familiar with cypress but at the time I dealt with some e2e tests I had the idea that always using wait with an absolute value to look for elements was not such a good practice.
Doesn't cypress already have a timeout/retry mechanism to look for elements? And couldn't this timeout be adjusted to our tests (I think this can be configured globally or individually)
Was searching for cypress documentation about this and maybe we don't need to use wait that much https://docs.cypress.io/guides/references/best-practices#Selecting-Elements 
Maybe for one tests it's not a big deal but for multiple ones it could get longer

Let me know what you think!